apps/demo/emails/codepen-challengers.tsx

```diff
@@ -0,0 +1,407 @@
+
+            <Text style={text}>
+              Last week, we kicked things off with round shapes. We "rounded" up
+              the Pens from week one in our{' '}
+              <Link style={blueLink}>#CodePenChallenge: Round</Link> collection.
+            </Text>
+
+            <Text style={text}>This week, we move on to cubes 🧊</Text>
+
+            <Text style={text}>
+              Creating cubes in the browser is all about mastery of illusion.
+              Take control of perspective and shadows and you can make the magic
+              of 3D on a flat screen 🧙
+            </Text>
+
+            <Text style={text}>
+              This week is a fun chance to work on your CSS shape-building
+              skills, or dig into a 3D JavaScript library like Three.js.
+            </Text>
+
+            <Text style={text}>
+              This week's starter template features an ice cube emoji to help
+              inspire a "cool" idea for your Pen. As always, the template is
+              just as jumping off point. Feel free to incorporate the 🧊 in your
+              creation, add more elements, or freeze it out completely and start
+              over from scratch!
+            </Text>
+
+            <Text style={yourChallenge}>
+              💪 <strong>Your Challenge:</strong>{' '}
+              <Link style={blueLink}>
+                create a Pen that includes cube shapes.
+              </Link>
+            </Text>
+
+            <Img
+              src={`${baseUrl}/static/codepen-cube.png`}
+              width={600}
+              alt="codepen"
+            />
+
+            <Section style={sectionPro}>
+              <Img
+                style={imagePro}
+                src={`${baseUrl}/static/codepen-pro.png`}
+                width={250}
+                alt="codepen"
+              />
+
+              <Text>
+                CodePen PRO combines a bunch of features that can help any
+                front-end designer or developer at any experience level.
+              </Text>
+
+              <Button style={button}>
+                <strong>Learn More</strong>
+              </Button>
+            </Section>
+          </Section>
+
+          <Text style={yellowSection}>
+            <strong>To participate:</strong>{' '}
+            <Link style={blueLink}>Create a Pen →</Link> and tag it{' '}
+            <Link style={blueLink}>
+              <strong>codepenchallenge</strong>
+            </Link>{' '}
+            and
+            <Link style={blueLink}>
+              {' '}
+              <strong>cpc-cubes</strong>
+            </Link>
+            . We'll be watching and gathering the Pens into a Collection, and
+            sharing on <Link style={blueLink}>Twitter</Link> and{' '}
+            <Link style={blueLink}>Instagram</Link> (Use the #CodePenChallenge
+            tag on Twitter and Instagram as well).
+          </Text>
+
+          <Section style={section}>
+            <Column style={ideas}>
```

Comment:
Column should be only used inside of `Row` otherwise the HTML structure will be wrong

Reply:
Thanks for taking a look into this @bruno88cabral, appreciate 🙏

Reply:
Done